en/api/configuration-generate.md

```diff
@@ -81,7 +81,7 @@ export default {
 ## fallback
 
 - Type: `String` or `Boolean`
-- Default: `'200.html'`
+- Default: `200.html` if left unset or false and `404.html` if set to true
```

Comment:
The outcome for different values is explained further down, so I'd suggest to keep it as is :relaxed: 	

```suggestion
- Default: `200.html`
```